rhizome/common/lib/util.rb

```diff
@@ -60,12 +60,14 @@ def sync_parent_dir(f)
 def safe_write_to_file(filename, content = nil)
   raise ArgumentError, "must provide either content or block" if (content.nil? && !block_given?) || (!content.nil? && block_given?)
   temp_filename = filename + ".tmp"
-  File.open(temp_filename, File::RDWR | File::CREAT) do |f|
-    f.flock(File::LOCK_EX)
+  File.open("#{temp_filename}.lock", File::RDWR | File::CREAT) do |lock_file|
```

Comment:
Does the lock file get deleted? If not, are lots of leftover lock files distracting?

Reply:
It does not. I changed the code to create the lock file at /tmp instead to ensure its deletion eventually.